.circleci/config.yml

```diff
@@ -1610,18 +1610,6 @@ workflows:
     jobs:
       *sdk_nightly_jobs
 
-  bucket_upload:
-    triggers:
-      - schedule:
-          # should trigger every day at 9 AM & 9 PM UTC (11:00 PM & 11:00 AM Israel Time)
-          cron: "0 9,21 * * *"
-          filters:
-            branches:
-              only:
-                - master
-    jobs:
-      *bucket_upload_jobs
-
   bucket_upload_trigger:
```

Comment:
Do we want to keep this?

Reply:
I think this falls under the category of all the remaining infrastructure we would need to still remove pending seeing that the flow operates in `GitLab` stably as @guyfreund mentioned in his comment https://github.com/demisto/content/pull/14130#pullrequestreview-727090083.